src/index.js

```diff
@@ -45,33 +90,58 @@ async function SaveUrl(context) {
   await crawlUrlByPython
     .then(async function(crawlingResults){
       await context.sendText(crawlingResults.title);
-      //[todo] get user information
+      let keyword = crawlingResults.keywords.split(“,”);
 
+      //[todo] get user information
+      let name = GetUserID(context)
 
       //[todo] save data to database
 
+      let con = mysql.createConnection({
```

Comment:
You’ll need a separated config file to store all sensitive data, e.g. db information and line access token.